examples/DateRangePickerWrapper.jsx

```diff
@@ -108,7 +111,14 @@ class DateRangePickerWrapper extends React.Component {
   }
 
   onDatesChange({ startDate, endDate }) {
-    this.setState({ startDate, endDate });
+    if (this.props.isPersianDateRangePicker) {
```

Comment:
Not a big deal, but maybe instead of this prop, momentJalaali could be passed in as a `stateDateWrapper` prop or something?

Reply:
So, You mean something like this? `stateDateWrapper={momentJalaali}`?
IDK, from my opinion it was cleaner way. What's the exact problem? In the naming of `isPersianDateRangePicker` prop?

Reply:
Persian might not be the only one this is needed for in the future; it seems like a more scalable approach.

Since it’s an example, it doesn’t matter much tho :-)

Reply:
what's your suggestion exactly?

Reply:
```jsx
const { stateDateWrapper } = this.props;
this.setState({ startDate: stateDateWrapper(startDate), endDate: stateDateWrapper(endDate) });
```

and then `stateDateWrapper` would default to `x => x`.

Reply:
Ok Thanks, 
but this line would be,
`this.setState({
      startDate: startDate && stateDateWrapper(startDate),
      endDate: endDate && stateDateWrapper(endDate),
  });`
because when we pass momentJalaali in as stateDateWrapper prop and each one of values was equal with null or undefined, we get error because is an invalid date.